code/game/objects/items/devices/radio/headset.dm

```diff
@@ -17,9 +32,24 @@
 
 /obj/item/radio/headset/examine(mob/user)
 	..()
-	to_chat(user, "<span class='notice'>To speak on the general radio frequency, use ; before speaking.</span>")
-	if (command)
-		to_chat(user, "<span class='notice'>Alt-click to toggle the high-volume mode.</span>")
+	
+	if(in_range(user, src))
+		// construction of frequency description
+		var/avail_chans = "Use ; for Common"
+		if(translate_binary && channels.len)
+			avail_chans += ", use :b for Binary"
+		else if(translate_binary)
+			avail_chans += ", and use :b for Binary"	
+		if(channels.len)
+			for(var/i = 1 to channels.len - 1)

```

Comment:
minor nit: the unofficial standard we have for for loops is generally `in` as in `for (var/i in 1 to channels.len-1)`, also `length(thing)` is faster than accessing `.len`

Reply:
Resolved in https://github.com/tgstation/tgstation/pull/40582/commits/0323bd68f4c82c7b2260ae2222f98d9443181f71